src/js/generate-payment-request.js

```diff
@@ -0,0 +1,39 @@
+payload.api_key = 'test_client_key_3btrkEyC6xvcByXLthuZx'
```

Comment:
I think this is the old way of setting up the api_key, check if this needs to be updated